src/tracker.rs

```diff
@@ -190,11 +190,11 @@ impl AckedIndexer for ProgressMap {
 /// which could be `Leader`, `Follower` and `Learner`.
 #[derive(Clone, Getters)]
 pub struct ProgressTracker {
-    progress: ProgressMap,
+    pub(crate) progress: ProgressMap,
 
     /// The current configuration state of the cluster.
     #[get = "pub"]
-    conf: Configuration,
+    pub(crate) conf: Configuration,
```

Comment:
Doesn't the above line states it's public accessible already?

Reply:
Using `conf()`, there will be a conflict between mutable borrow of `self.prs.progress` and immutable borrow of `self.prs.conf`.